src/odb/src/db/dbProperty.h

```diff
@@ -73,13 +76,7 @@ class _dbProperty : public _dbObject
   dbId<_dbProperty> _next;
   uint _owner;
 
-  union
-  {
-    char* _str_val;
-    uint _bool_val;
-    int _int_val;
-    double _double_val;
-  } _value;
+  std::variant<std::string, uint, int, double> _value;
```

Comment:
While you are cleaning this up perhaps you could use bool rather than uint for a boolean property.